### ISSUE TITLE
[Feature Request] Extend the list of available parameters you allow the customers to configure via AKS Autoscaler profile

### ISSUE THREAD

**Issue by lawrencewilson** [Azure/AKS#1559]
**What happened**:

We want to switch over to using the the Azure AKS Cluster Autoscaler from a self-hosted Cluster Autoscaler. 
The property that we want to set is `--skip-nodes-with-local-storage=false` which is not available to be configured using the AKS autoscler profile. 
We also can't be sure what the default values of the AKS Cluster autoscaler is, and if it will change in a future version of AKS autoscaler

**What you expected to happen**:
This command should accept `skip-nodes-with-local-storage=false`
```
az aks update \
  --resource-group myResourceGroup \
  --name myAKSCluster \
  --cluster-autoscaler-profile scan-interval=30s skip-nodes-with-local-storage=false
```

**Anything else we need to know?**:

We believe that the current default in AKS autoscaler profile is `skip-nodes-with-local-storage=false`. Which is great, exactly what we want it to be but how can we know for sure this will not change?

**Environment**:
- Kubernetes version (use `kubectl version`): 1.15.10
- Size of cluster (how many worker nodes are in the cluster?): 10
- General description of workloads in the cluster: software application

**Comment by palma21** [comment]
Hi @lawrencewilson you are correct, that is the default, we don't plan to change any of the defaults. And if that ever is justified, we will follow breaking change procedure and communicate via service announcements (service health notification, email, release notes announcements) before that change is merged. In that occurrence we'd likely make those parameters mutable to avoid breaking any configuration.

I think you raise a good point on documenting all the defaults and not just the ones we expose as mutable. I will follow up to add that to the docs as such.

Do you think this would meet your needs?

**Comment by marwanad** [comment]
This should be available with the [September API](https://docs.microsoft.com/en-us/rest/api/aks/managedclusters/createorupdate#autoscalerprofile).

For clients, it's available in CLI 2.12.0, CLI-extensions 0.4.63 and via ARM templates.

Feel free to report back on this thread if you hit issues.
